server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
I am ok with introducing new column, but there are no real reasons to change the creation_time (creation_time is only set once for a new record and should never be updated), so I suggest leaving that column the way it was.

Apart from that I'd like to get confirmation from Cliff that he really wants to log every update to the _project_  vs every update to the _model_ :-}